### PR TITLE
Feature/1 wrap elements

### PR DESCRIPTION
**Pull request by anacidre**

### CODE REVIEW

File: src/app/app.component.html [hunk 1]
```diff
@@ -1 +1,3 @@
-<bb-card></bb-card>
\ No newline at end of file
+<section class="component__grid">
+  <bb-card class="component__container" *ngFor="let berry of berries" [berry]="berry"></bb-card>
```
Comment (orestes) [inline src/app/app.component.html:2]: Much better, I can see the list now.

File: src/app/bb-card/bb-card.component.html [hunk 2]
```diff
@@ -1,12 +1,8 @@
...
-			<h3 class="product__title" aria-label="product name">
-        {{ berry.title }}</h3>
-			<p class="product__description">{{ berry.description }}</p>
-			<p class="product__price" aria-label="product price">
-        {{ berry.price |currency:'EUR' }} / kilo</p>
-		</div>
-	</div>
-</section>
\ No newline at end of file
+<img class="product__img" [src]="berry.img">
```
Comment (orestes) [inline src/app/bb-card/bb-card.component.html:1]: You changed this already, nice!

Comment (anacidre) [reply src/app/bb-card/bb-card.component.html:1]: Yep! Binding ftw

File: src/app/app.component.css [hunk 3]
```diff
@@ -1,3 +1,15 @@
...
+  max-width: 800px;
+}
+.component__container {
+  display: flex;
+  flex-direction: column;
+  flex-grow: 1;
+  width: 50%;
+  max-width: 200px;
+  background-color: #BF395B;
+  margin: 10px;
```
Comment (orestes) [inline src/app/app.component.css:14]: Margin in px? Uuuh...

**Review by orestes — CHANGES_REQUESTED** [review]

**Review by orestes — APPROVED** [review]

**Pull request merged**